front/src/applications/stdcm/components/StdcmResults/SendToRailwayManagerModal.tsx

```diff
@@ -0,0 +1,437 @@
+      {
+        label: t('modal.totalTonnage'),
+        value: consist?.totalMass ? `${consist.totalMass} t` : '-',
+      },
+      { label: t('modal.rollingStock'), value: consist?.towedRollingStock?.name },
+      { label: t('modal.referenceEngine'), value: stdcmData.rollingStock.name ?? '-' },
+      {
+        label: t('modal.maxSpeed'),
+        value: consist?.maxSpeed != null ? `${consist.maxSpeed} km/h` : '-',
+      },
+      {
+        label: t('modal.totalLength'),
+        value: consist?.totalLength != null ? `${consist.totalLength} m` : '-',
+      },
+      { label: t('modal.gauge'), value: consist?.loadingGauge ?? '-' },
+    ],
+    [consist, stdcmData.rollingStock.name, t]
+  );
+
+  const pathTypeOptions = [
+    { value: 'LongFRET', label: t('modal.LongFRET') },
+    { value: 'Autre', label: t('modal.Autre') },
+  ];
+
+  const pdfDocument = useMemo(
+    () => (
+      <StdcmSimulationReportSheet
+        stdcmLinkedTrains={linkedTrains}
+        stdcmData={stdcmData}
+        consist={consist}
+        simulationReportSheetNumber={simulationReportSheetNumber}
+        operationalPointsList={operationalPointsList}
+        simulationSheetLogo={simulationSheetLogo}
+        similarTrains={similarTrains}
+      />
+    ),
+    [
+      linkedTrains,
+      stdcmData,
+      consist,
+      simulationReportSheetNumber,
+      operationalPointsList,
+      simulationSheetLogo,
+      similarTrains,
+    ]
+  );
+
+  const handleCsChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    const input = e.target.value.toUpperCase();
+    setCsCode(input);
+  };
+
+  const handleCommentChange = (e: React.ChangeEvent<HTMLTextAreaElement>) => {
+    const transformedValue = e.target.value
+      .replace(/&/g, 'et') // replace "&" with "et"
+      .replace(/</g, '('); // replace "<" with "("
+
+    setComment(transformedValue);
+  };
+
+  const handleBeforeChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    const value = Number(e.currentTarget.value);
+    if (!validateTolerance(value)) return;
+    setBeforeTolerance(value);
+  };
+
+  const handleAfterChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    const value = Number(e.currentTarget.value);
+    if (!validateTolerance(value)) return;
+    setAfterTolerance(value);
+  };
+
+  const showPathTypeError = useCallback(() => {
+    setPathTypeError(true);
+    setTimeout(() => setPathTypeError(false), 500);
+  }, []);
+
+  const handleCreate = async () => {
+    if (!pathType) {
+      showPathTypeError();
+      return;
+    }
+
+    if (!validateTolerance(beforeTolerance) || !validateTolerance(afterTolerance)) {
+      return;
+    }
+
+    const instance = pdf(pdfDocument);
+    const blobToUpload = await instance.toBlob();
+
+    const filename = `${'Last Minute Request'}-${simulationReportSheetNumber}`;
+
+    const formData = new FormData();
+
+    const normalizedDate = realConstraintTime;
```

Comment:
Sorry, my mistake this morning
```suggestion
    const normalizedDate = realDepartureTime;
```

Reply:
Done